Pikipick/Pikipick/PresentationView.swift

```diff
@@ -13,43 +13,48 @@ struct PresentationView: View {
     @State private var showingAlert = false
     
     var body: some View {
-        VStack{
-            Spacer()
-            ButtonActionView()
-        }
-        .onAppear(perform: {
-            AppUtility.lockOrientation(UIInterfaceOrientationMask.landscape)
-            UIDevice.current.setValue(UIInterfaceOrientation.landscapeLeft.rawValue, forKey: "orientation")
-            UINavigationController.attemptRotationToDeviceOrientation()
-        })
-        .navigationBarBackButtonHidden(true)
-        .toolbar {
-            ToolbarItem(placement: .navigationBarLeading) {
-                Button {
-                    showingAlert = true
-                }
-            label: {
-                Image(systemName: "chevron.backward")
-                Text("Back")
+        ZStack{
+            Color.backgroundColor.ignoresSafeArea()
+            VStack{
+                Spacer()
+                ButtonActionView()
+                    .offset(y: -17)
             }
+            .onAppear(perform: {
+                AppUtility.lockOrientation(UIInterfaceOrientationMask.landscape)
+                UIDevice.current.setValue(UIInterfaceOrientation.landscapeLeft.rawValue, forKey: "orientation")
+                UINavigationController.attemptRotationToDeviceOrientation()
+            })
+            .navigationBarBackButtonHidden(true)
+            .toolbar {
+                ToolbarItem(placement: .navigationBarLeading) {
+                    Button {
+                        showingAlert = true
```

Comment:
```suggestion
                    Button {
                        showingAlert.toggle()
```

단순 대입이 아닌 toggle을 써도 좋을 것 같아요!

Reply:
수정했습니다!!